packages/slate-editor/src/modules/editor/getEnabledExtensions.ts

```diff
@@ -448,12 +448,13 @@ function buildPlaceholdersExtensionConfiguration({
                     withEmbeds && withPlaceholders && withPlaceholders.withPastedUrlsUnfurling
                         ? {
                               isAllowed(editor) {
-                                  const isSelectionEmpty = EditorCommands.isSelectionEmpty(editor);
+                                  const isSelectionInEmptyParagraph =
+                                      EditorCommands.isCursorInEmptyParagraph(editor);
                                   const isInsideTable = TablesEditor.isTablesEditor(editor)
                                       ? TablesEditor.isInTable(editor)
                                       : false;
 
-                                  return isSelectionEmpty && !isInsideTable;
+                                  return isSelectionInEmptyParagraph && !isInsideTable;
```

Comment:
@e1himself I've removed the `isSelectionEmpty` call since `isCursorInEmptyParagraph` already makes sure range is not expanded (therefore collapsed).

I haven't noticed any difference in behaviour, so should be okay 🙂

Reply:
IDK, I'd personally keep it for clarity.

Reply:
Yeah, makes sense, I've reverted it, thanks!